debian/changelog

```diff
@@ -1,8 +1,14 @@
+kano-profile (2.2-1) unstable; urgency=low
```

Comment:
It would be great to transition to a format of X.Y.Z-B for package versions, to accommodate the urgent updates scheme (and to be consistent across packages). 
Do you mind changing the version to `2.2.0-1`?

It is not really necessary to do this in this PR, if you can't be bothered, we could open a new one after this has merged.